uml/sequence_diagrams/Settings.txt

```diff
@@ -0,0 +1,35 @@
+@startuml
+participant DisplayViews 
+participant SettingsView
+participant Storage
+participant BlueTooth
+participant Brightness
+participant HealtStatusCheck
+participant DisplayError 
+
+opt Settings
+  DisplayViews --> SettingsView : GoTo Settings
+end
+opt BlueTooth
+  SettingsView --> BlueTooth: change BlueTooth ON/OFF

```

Comment:
Regarding the bluetooth pairing sequence in `bluetooth.png` makes sense, should we express it also in a sequence diagram, of have an Activity diagram for that ?

I can add an Activity diagram and maybe we can see if it is sufficient...

Reply:
I will add a sequence diagram for this.

We should do an activity diagram also  for this and power management.

Reply:
I added an actvity diagram for bluetooth pairing here: https://github.com/markroche92/sw-eng-smart-watch/pull/16

Reply:
On second thought, I don't see any point in a sequence diagram for bluetooth.  It is best expressed by the activity diagram and state diagram. A sequence diagram would appear a little clumsy.